addons/l10n_mn/data/account.account.template.csv

```diff
@@ -6,39 +6,43 @@ account_template_1103_0101,11030101,Хугацаагүй хадгаламж,asse
-account_template_1201_0202,12010202,Худалдааны авлага Хувь хүн (PoS),asset_receivable,account_tag_operating_product,1,mn_chart_1
-account_template_1201_0301,12010301,Худалдааны авлага Холбоотой тал,asset_receivable,account_tag_operating_product,1,mn_chart_1
-account_template_1202_0101,12020101,Авлагын бичиг,asset_current,account_tag_operating_product,0,mn_chart_1
-account_template_1203_0101,12030101,Найдваргүй авлагын хасагдуулга,asset_current,account_tag_operating_product,0,mn_chart_1
-account_template_1204_0101,12040101,Урьдчилж төлсөн ААНОАТатвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0201,12040201,Урьдчилж төлсөн ХХОАТатвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0301,12040301,Урьдчилж төлсөн НӨАТатвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0401,12040401,Урьдчилж төлсөн Гаалийн албан татвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0501,12040501,Урьдчилж төлсөн Үл хөдлөх хөрөнгийн албан татвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0601,12040601,Урьдчилж төлсөн Газрын төлбөр,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0701,12040701,Урьдчилж төлсөн АТӨЯХАТатвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0801,12040801,Урьдчилж төлсөн Агаарын бохирдлын татвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_0901,12040901,Урьдчилж төлсөн Онцгой албан татвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_1001,12041001,Урьдчилж төлсөн Байгалийн нөөц ашиглалтын төлбөр,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1204_9901,12049901,Урьдчилж төлсөн Бусад татвар,asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1205_0101,12050101,"Урьдчилж төлсөн ЭМ, НДШ",asset_current,account_tag_operating_taxrefund,0,mn_chart_1
-account_template_1205_0201,12050201,Урьдчилж төлсөн Сайн дурын даатгалын хураамж,asset_current,account_tag_operating_insurance2,0,mn_chart_1
-account_template_1206_0101,12060101,"Цалингийн урьдчилгаа, богино хугацаат зээл",asset_receivable,account_tag_operating_employee,1,mn_chart_1
-account_template_1206_0201,12060201,"Төлбөр, дутагдал",asset_receivable,account_tag_operating_employee,1,mn_chart_1
+account_template_1111_0101,11110101,Мөнгөн гүйлгээний хүлээлгийн түр данс,asset_current,,0,l10n_mn.mn_chart_1
+account_template_1201_0101,12010101,Худалдааны авлага ААН,asset_receivable,account_cashflow_tag_123,1,mn_chart_1
+account_template_1201_0201,12010201,Худалдааны авлага Хувь хүн,asset_receivable,account_cashflow_tag_123,1,mn_chart_1
+account_template_1201_0202,12010202,Худалдааны авлага Хувь хүн (PoS),asset_receivable,account_cashflow_tag_123,1,mn_chart_1
```

Comment:
Thanks so much for your changes to the Chart of Accounts!
I think they are quite sensible. The only concern that I had was regarding the Cash Flow statement, because, as I mentioned in our call, it works differently from the generic cash flow statement (which takes into account reconciliations). I think you mentioned that you would discuss with your accountants what exactly is needed for the Cash Flow Statement, and I'm looking forward to hearing back from you about that.